cluster/azure/tm-endpoint-ip/variables.tf

```diff
@@ -0,0 +1,49 @@
+variable "traffic_manager_profile_name" {
+  type = "string"
+}
+
+variable "traffic_manager_resource_group_name" {
+  type = "string"
+}
+
+variable "public_ip_name" {
+  type = "string"
+}
+
+variable "endpoint_name" {
+  type = "string"
+}
+
+variable "resource_group_name" {
+  type = "string"
+}
+
+variable "resource_location" {
+  type = "string"
+}
+
+variable "ip_address_filename" {
+  type    = "string"
+  default = "bedrock_public_ip_address"
+}
+
+variable "ipaddress_to_disk" {
+  description = "This disables or enables the ip address output file from being written to disk."
+  type        = "string"
+  default     = "true"
```

Comment:
My vote is to set this to false by default so we don't have the linux requirement & processing time for it. However, this information is useful, so we should include the output for the public IP so people can still input to their [Kubernetes yaml](https://docs.microsoft.com/en-us/azure/aks/static-ip#create-a-service-using-the-static-ip-address).

Reply:
that makes sense and I agree that it makes finding ip address easier for configuration of load balancer yaml.